__tests__/app.test.js

```diff
@@ -94,3 +94,48 @@ describe("GET /api", () => {
         });
     });
 })
+
+describe("GET /api/reviews", () => {
+    test.only("status:200, responds with an array of all the review objects with the correct properties, sorted in descending order", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then((response) => {
+            console.log(response.body.reviews)
+            response.body.reviews.forEach((review) => {
```

Comment:
Please add length check here so you can be sure you're returning the correct amount of reviews